srcts/src/utils/index.ts

```diff
@@ -178,7 +178,7 @@ function scopeExprToFunc(expr: string): (scope: unknown) => boolean {
     throw e;
   }
 
-  return function (scope: unknown): boolean {
+  return function (scope: unknown): unknown {
     return func.call(scope);
```

Comment:
Can we revert the type / `!!` changes and instead force a boolean here?

```suggestion
    return Boolean(func.call(scope));
```

Reply:
Hmmmm. But the name of the function doesn't point to bool.

Ok